.gitignore

```diff
@@ -14,3 +14,4 @@ stats.json
 web-components/screenshots/Chrome/failed/*
 server.crt
 server.key
+milo_artifacts
```

Comment:
`milo-libs` sounds a little bit nicer to me :)

Reply:
agree